<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUser</class>
 <widget class="QWidget" name="NewUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册用户</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/ui/new</normaloff>:/ui/new</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>账号:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_id">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>密码:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_passwords">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>确认密码:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_passwords2">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_new">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>注册</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_cancel">
         <property name="font">
          <font>
           <family>Adobe Arabic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
